js/ui/dash.js

```diff
@@ -343,7 +343,8 @@ const DashActor = new Lang.Class({
         let layout = new Clutter.BoxLayout({ orientation: Clutter.Orientation.VERTICAL });
         this.parent({ name: 'dash',
                       layout_manager: layout,
-                      clip_to_allocation: true });
+                      clip_to_allocation: true,
+                      visible: false });
```

Comment:
Actually, I just gave it a try now and the change seems to be doable:
```
 js/perf/core.js           |   2 --
 js/perf/hwtest.js         |   1 -
 js/ui/appDisplay.js       |   6 +++---
 js/ui/dash.js             |   2 --
 js/ui/iconGrid.js         | 110 --------------------------------------------------------------------------------------------------------------
 js/ui/overview.js         |  12 ------------
 js/ui/overviewControls.js |  37 ++-----------------------------------
 js/ui/viewSelector.js     |  39 +++++----------------------------------
 8 files changed, 10 insertions(+), 199 deletions(-)
```
Maybe it's one of those things that looked messier and harder to me one month ago, out of pure ignorance. I'll keep addressing your comments and rebasing things based on this for now, see how it turns out...